app/views/spam2/_queue.html.erb

```diff
@@ -65,7 +65,8 @@ $(document).ready(function () {
         <% @queue.each do |node| %>
           <tr id="n<%= node.id %>">
             <td><input  class="selectedId" value="<%= node.nid %>" type="checkbox" /></td>
-            <td style="width:250px !important;">
+            <td style="width:250px !importan<th style="width:250px !important;"><%=translation('users.profile.title')%></th>
+									      t;">
```

Comment:
@sandipanpanda Can you see the error here? We need to fix it before we merge this, otherwise the page won't render properly.

You'll need to revert the changes, then make another commit to this pull request. Let me know if you have any questions about that!